qtensor/optimisation/kahypar_ordering/generate_TN.py

```diff
@@ -24,3 +24,9 @@ def circ2tn(circuit: List[qtree.operators.Gate]):
     TN = dual_hg(dual_tn)
     return TN
 
+def tn2tn(tn): #tn: qtensor.optimization.QtreeTensorNet
+    #tn = qtensor.optimisation.QtreeTensorNet.from_qtree_gates(circuit)
```

Comment:
1. Yes, I recover it to the ones in the dev branch. Is it correct now?
2. Yes, I put images on purpose since I thought they are not very large. I can remove them if they are not necessary.
3. Thanks. I added a type hint for it.

Reply:
1. Yes looks like it's fine. 
2. Ok. By the way, even if you delete them now, they are still a part of the repository forever:)
3. I don't see it. Your latest commit is 0c579f3